bbl/i18n/uk/OrcaSlicer_uk.po

```diff
@@ -2357,17 +2357,18 @@ msgid ""
 "Please make sure whether to use the temperature to print.\n"
 "\n"
 msgstr ""
-"Форсунка може бути заблокована, коли температура виходить за "
-"межірекомендованого діапазону.\n"
+"Форсунка може бути заблокована, коли температура виходить за межі "
+"рекомендованого діапазону.\n"
 "Переконайтеся, що для друку використовується температура.\n"
+"\n"
 
 #, c-format, boost-format
 msgid ""
 "Recommended nozzle temperature of this filament type is [%d, %d] degree "
 "centigrade"
 msgstr ""
-"Рекомендована температура сопла для цього типу нитки становить [%d, "
-"%d]градусів Цельсія"
+"Рекомендована температура сопла для цього типу нитки становить [%d, %d] "
```

Comment:
Use one translation for nozzle everywhere. L2360 uses different translation than current

Reply:
Thanks for joining!

Reply:
Missed that one, thank you! Fixed in a6dcadc5198ff3d6449d3a6c32224e25c0c45a45